sql/flow.go

```diff
@@ -80,7 +80,12 @@ func CommonDockerUtil(cmd, args []string, flags map[string]string, mountDirs []s
 		return err
 	}
 
-	dockerfileContent := []byte(fmt.Sprintf(include.Dockerfile, PythonVersion, astroSQLCliVersion))
+	baseImage, err := getBaseDockerImageURI(astroSQLCliConfigURL)
```

Comment:
nit: having `astroSQLCliConfigURL` constant defined in this file itself instead of `sql/flow.go` might be better for code readability, wdyt?